src/components/molecules/EmailPrint.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { EmailPrintButton } from "../atoms/EmailPrintButton";
+
+/**
+ * This component layout the email and print buttons
+ */
+export function EmailPrint(props) {
+  const handleClick = (event) => props.onClick(event.currentTarget.id);
+  return (
+    <div className="md:flex">
+      <EmailPrintButton
+        id="email"
+        icon="icon-envelop"
+        text={props.emailText}
+        onClick={handleClick}
+      />
+      <div className="md:m-6" />
```

Comment:
Why is this div here? You should never have unnecessary elements to achieve the styling you want.